pkgs/development/tools/f2c/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation {
+  pname = "f2c";
+  version = "20200916";
+
+  src = fetchurl {
+    url = "http://www.netlib.org/f2c/src.tgz";
+    sha256 = "0d8xfbv6dk4dz95qds7sd44b5hvara07f2g2c5g4xiwim9b7916l";
+  };
+
+  makeFlags = [ "-f" "makefile.u" ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin $out/share/man/man1
+    install -m755 f2c $out/bin
+    install -m755 xsum $out/bin
+    install f2c.1t $out/share/man/man1
```

Comment:
```suggestion
    mkdir -p $out/bin
    install -m755 f2c $out/bin
    install -m755 xsum $out/bin
    installManPage f2c.1
```

Reply:
This unfortunately garbles the man page by forcing it to 80 chars width (which makes it pretty hard to read).

Reply:
I don't think man pages can have a 1t extensions.

Reply:
This may not be very common, but it works just fine (and it is how it is delivered by the project). 
`man f2c` works.

Reply:
Searched the nix-store out of curiosity: it is still not very common but can be found for example in `ncurses` (`*.3x`, `*.1m`) or `libtirpc` (`*.3t`).